scripts/export_container_bundle

```diff
@@ -25,62 +27,44 @@ while getopts "hc:o:" opt; do
     h)
       print_usage_and_exit;;
     o)
-      readonly OUTPUT_FILE="${OPTARG}";;
+      readonly OUTPUT_OCI_BUNDLE_TAR="${OPTARG}";;
     c)
-      readonly CONTAINER_IMAGE="${OPTARG}";;
+      readonly INPUT_OCI_IMAGE_TAR="${OPTARG}";;
     *)
       echo "Invalid argument: ${OPTARG}"
       exit 1;;
   esac
 done
 
-if [ -z "${CONTAINER_IMAGE}" ]; then
-  echo "Missing required option: -c <container-image>"
+if [ -z "${INPUT_OCI_IMAGE_TAR}" ]; then
```

Comment:
Could you use `[[` instead of `[` since you are here?

See http://go/shell-style#tests